packages/core/test/unit/dialects/mysql/query.test.js

```
@@ -33,4 +35,69 @@ describe('[MYSQL/MARIADB Specific] Query', () => {
+        { timestamps: false },
+      );
+    });
+
+    // MariaDB's driver returns insertId as a BigInt (the range arithmetic depends on it);
+    // MySQL's returns a plain number. Otherwise the two paths behave identically.
+    const dialects = [
+      { name: 'MySqlQuery', Query: MySqlQuery, insertId: 10, id: n => n },
+      { name: 'MariaDbQuery', Query: MariaDbQuery, insertId: 10n, id: n => BigInt(n) },
+    ];
+
+    for (const { name, Query, insertId, id } of dialects) {
```

ESLint

[error] Expected object keys to be in sorted order. Expected insertId to be before Query.

(sort-destructure-keys/sort-destructure-keys)